deploy/docker/notebook/values.yaml

```diff
@@ -488,6 +487,14 @@ addfiles:
                           return True 
                   
                   return False
+
+              def script_get_file_id(self, os_path):
+                  try:
+                      r = requests.get(self.retrieve_uri, params={"email": os.getenv("JUPYTERHUB_USER"), "path": os_path}, timeout=1)
+                  except Exception as e:
+                      logging.getLogger().error(f"Failed to retrieve {os_path} from Notebooks Hub", exc_info=e)
+                  file_id = r.json()[0]["_id"]
```

Comment:
We should add a ternary condition to check for length here since the length of the file data array could be zero in the case of a file not being initialized in the database before attempting to rename or delete it.

file_id = r.json()[0]["_id"] if len(r) > 0 else '0'